app/src/v18/java/com/app/missednotificationsreminder/service/AbstractReminderNotificationListenerService.java

```diff
@@ -84,36 +93,35 @@ public boolean checkNotificationForAtLeastOnePackageExists(Collection<String> pa
         boolean result = false;
         StatusBarNotification[] activeNotifications = getActiveNotifications();
         List<String> activeNotificationKeys = new ArrayList<>();
+        Timber.d("checkNotificationForAtLeastOnePackageExists: %1$d notifications", activeNotifications.length);
```

Comment:
Can you please move this one line below the activeNotifications null check or add null check in the line itself

Reply:
Looks like you've done it already. Thanks.